db/schema.rb

```diff
@@ -371,6 +371,7 @@
     t.string "wallet_provider_id"
```

Comment:
I think the `db/schema.rb` needs to be rebased too

Reply:
This is on the potential_payments table not the publishers table 👍